lib/brakeman.rb

```diff
@@ -180,6 +180,8 @@ def self.get_formats_from_output_format output_format
       [:to_json]
     when :markdown, :to_markdown
       [:to_markdown]
+    when :cc, :codeclimate, :to_codeclimate
```

Comment:
Yes, I think that would be helpful. I've added the branch to `get_formats_from_output_files`. I've also fixed the `:cc` option – thanks!